### PR TITLE
added custom login theme

### PR DESCRIPTION
**Pull request by roy-sagar11**
## Summary
Added custom login page in keycloak

## Screenshots (if applicable)

Screen Shot 2022-06-06 at 2 50 10 PM

### CODE REVIEW

File: keycloak/extensions/themes/src/main/resources/theme/ssag/login/template.ftl [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  <div class="ssag-header">
+      <div class="ssag-logo-wrapper">
+          <img class ="ssag-logo" src="${url.resourcesPath}/img/logo-banner.svg" alt="BC Gov logo"/>
+      </div>
+      <div class="ssag-title-wrapper" >
+          <div class="ssag-bottom-title" >
+              ${client.getDescription()!""}
+          </div>
+      </div>  
+  </div>
```
Comment (simensma-fresh) [inline keycloak/extensions/themes/src/main/resources/theme/ssag/login/template.ftl]: I'm wondering if we could add in a couple of the other missing sections here as well from https://github.com/keycloak/keycloak/blob/9c01d819cb8d1709503e61db62f630686d5a1c5e/themes/src/main/resources/theme/base/login/template.ftl:
- Error/warning section:
- "Try another way link" 
- "Displayinfo"

File: keycloak/extensions/themes/src/main/resources/theme/ssag/login/template.ftl [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  <div class="ssag-page">
+    <div>
+      <h3 class="ssag-title">Log in to ${client.getDescription()!''}</h3>
+    </div>
+    <div class="ssag-login ssag-card">
+      <div class="ssag-container">
+        <p class="ssag-subheader">Please select your preferred login method</p>
+        <#nested "socialProviders">
```
Comment (simensma-fresh) [inline keycloak/extensions/themes/src/main/resources/theme/ssag/login/template.ftl]: Could we do something similar to https://github.com/bcgov/social-access-portal/blob/main/keycloak/theme/bcgov-idp-stopper/login/login.ftl here? As the `login.username?contains("##" + p.alias + "##")` determines what identity providers should show up as set in https://github.com/bcgov/social-access-portal/blob/main/keycloak/extensions/services/src/main/java/com/github/bcgov/keycloak/authenticators/browser/IdentityProviderStopForm.java

**Review by simensma-fresh — CHANGES_REQUESTED** [review]
Looks good! Just a couple of changes we'll need to match the existing `bcgov-idp-stopper` theme.

We'll also need to change the `login_theme`s in our tf config from `bcgov-idp-stopper` to `ssag` to apply this in dev

File: keycloak/extensions/themes/src/main/resources/theme/ssag/login/resources/css/styles.css [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+  font-weight: 400;
+  src: url("../font/BCSans-Regular.woff") format("woff"), url("../font/BCSans-Regular.ttf") format("ttf");
+}
+
+@font-face {
+  font-family: "BCSans-bold";
+  font-style: normal;
+  font-weight: 700;
+  src: url("../font/BCSans-Bold.woff") format("woff");
+}
```
Comment (iman-jamali-fw) [inline keycloak/extensions/themes/src/main/resources/theme/ssag/login/resources/css/styles.css]: Not sure how much value it adds, but it is possible to add one font-face with multiple weights and styles. Please see [here](https://stackoverflow.com/questions/28279989/multiple-font-weights-one-font-face-query). Feel free to try if that seems reasonable for this project.

**Review by iman-jamali-fw — APPROVED** [review]
LGTM

File: keycloak/extensions/themes/src/main/resources/theme/ssag/login/resources/css/styles.css (hunk 3 above)
Comment (roy-sagar11) [reply keycloak/extensions/themes/src/main/resources/theme/ssag/login/resources/css/styles.css]: good point. will do that!

File: keycloak/extensions/themes/src/main/resources/theme/bcgov/login/resources/css/styles.css [hunk 4]
```diff
@@ -1,8 +1,34 @@
...
+  font-family: BCSans;
+  font-weight: normal;
+  src: url("../font/BCSans-Regular.woff") format("woff");
+}
+
+@font-face {
+  font-family: BCSans;
+  font-weight: bold;
+  src: url("../font/BCSans-Bold.woff") format("woff");
+}
```
Comment (iman-jamali-fw) [inline keycloak/extensions/themes/src/main/resources/theme/bcgov/login/resources/css/styles.css:11]: Looks great! Much cleaner now 👍🏼

**Review by iman-jamali-fw — APPROVED** [review]
Changes look good to me

**Review by simensma-fresh — APPROVED** [review]

**Pull request merged**